browsers/chrome.json

```diff
@@ -588,19 +588,26 @@
         "85": {
           "release_date": "2020-08-25",
           "release_notes": "https://chromereleases.googleblog.com/2020/08/stable-channel-update-for-desktop_25.html",
-          "status": "current",
+          "status": "retired",
           "engine": "Blink",
           "engine_version": "85"
         },
         "86": {
-          "status": "beta",
+          "release_date": "2020-10-20",
+          "release_notes": "https://chromereleases.googleblog.com/2020/10/stable-channel-update-for-desktop_20.html",
+          "status": "current",
           "engine": "Blink",
           "engine_version": "86"
         },
         "87": {
-          "status": "nightly",
+          "status": "beta",
```

Comment:
Yep, https://omahaproxy.appspot.com/ confirms 87 is beta now. (The channels don't always change in sync.)